Makefile

```diff
@@ -53,6 +53,9 @@ view-submission: ${TEX_MAIN_SUBMISSION} ${IMAGES}
 
 clean:
 	latexmk -C
+	[ -d "_minted" ] && rm -r _minted || true
+	[ -d "_minted-$(basename $(TEX_MAIN_PAPER) .tex)" ] && rm -r "_minted-$(basename $(TEX_MAIN_PAPER) .tex)" || true
```

Comment:
do you need the `.tex` here and below? plain `basename` should work, no?

Reply:
yep